### PR TITLE
feat: Update manifest to add acceptFromFlagship

### PR DESCRIPTION
**Pull request by acezard**
### ✨ Features

* Update manifest to add acceptFromFlagship
 
* Also add placeholder route that will handle mobile uploads later

### CODE REVIEW

File: src/drive/web/modules/views/Create/CreateFromFlagship.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```
Comment (Crash--) [inline src/drive/web/modules/views/Create/CreateFromFlagship.jsx:1]: should be more in `upload` than `Create`.

File: src/drive/targets/manifest.webapp [hunk 2]
```diff
@@ -175,5 +175,12 @@
       "verbs": ["POST"],
       "description": "Remote-doctype required to send anonymized measures to the DACC shared among mycozy.eu's Cozy."
     }
+  },
+  "accept_from_flagship": true,
+  "accept_documents_from_flagship": {
+    "accepted_mime_types": ["application/octet-stream"],
```
Comment (Crash--) [inline src/drive/targets/manifest.webapp:181]: this will not be the only accepted mime. In fact, with Drive, you can go with : `[*]`.

Comment (acezard) [reply src/drive/targets/manifest.webapp:181]: Updated with "*/*" according to https://www.rfc-editor.org/rfc/rfc7231#section-5.3.2

File: src/drive/web/modules/views/Create/CreateFromFlagship.jsx (hunk 1 above)
Comment (acezard) [reply src/drive/web/modules/views/Create/CreateFromFlagship.jsx]: updated, will replace 
```javascript
      {__TARGET__ === 'mobile' && (
        <Route path="uploadfrommobile" element={<UploadFromMobile />} />
      )}
 ```
 in subsequent pull request

**Comment by acezard** [comment]
closing this pull request as it is now redundant with https://github.com/cozy/cozy-drive/pull/2969

**Pull request closed without merging**